### DISCUSSION TITLE
Redhat 7/8 - Xorg starts X session on physical monitor/console

### DISCUSSION THREAD

**Discussion by effectivelywild** [neutrinolabs/xrdp#2283] [Q&A]
Hello,

This has been tested on Redhat 7/8 using GNOME with the same result. 

I cannot get the Xorg (code 20) session to work. When I make a connection it creates an X session which appears on the console/monitor of the host then the RDP client eventually times out.

Subsequent connection attempts try to connect to that same session that was launched and timeout.

Is there documentation that describes the connection logic and what exactly should be happening when I try to use Xorg?

Thanks

**Comment by matt335672** [comment]
This is a little odd:-
> When I make a connection it creates an X session which appears on the console/monitor of the host then the RDP client eventually times out.

With the default settings, xrdp should use display :10 for a first connection. Yours seems to be using display :0

Can you check that `X11DisplayOffset` in `/etc/xrdp/sesman.ini` is set to 10?

Also, after the connection attempt, can you post the last 20 lines of so from `/var/log/xrdp-sesman.log`? The timestamps should make it obvious what to post.
